org.eclipse.wb.core/src/org/eclipse/wb/core/controls/palette/DefaultPalettePreferences.java

```diff
@@ -19,19 +20,31 @@
  * @coverage core.control.palette
  */
 public final class DefaultPalettePreferences implements IPalettePreferences {
+  @Override

```

Comment:
Cool but unrelated. Can you send a new PR with the cleanup add @Override for the whole plug-in?